scri/flux.py

```diff
@@ -437,12 +438,312 @@ def angular_momentum_flux(h, hdot=None):
     return jdot
 
 
+""" Boost flux is analytically calculated using the Wald & Zoupas formalism.
+Flanagan and Nichols [1510.03386] is referred for the calculation.
+Then, the GHP formalism is implemented to make it computationally feasible.
+The conventions of Ruiz+ (2008) [0707.4654] are followed.
```

Comment:
The reason I used Ruiz et.al because the other fluxes were defined using that convention. Also, when I say GHP formalism I just mean the spinor formalism that involves using spinors and spinor derivatives to do calculations. Yeah, I am not using the convention from the GHP paper. I just wanted to be consistent with the previous definitions of fluxes.